app/views/fact_values/_fact.html.erb

```diff
@@ -13,11 +13,11 @@
   </td>
   <td class="fact-col">
     <% unless fact.compose %>
-      <%= link_to trunc_with_tooltip(fact.value, 40),
+      <%= link_to truncate(fact.value, :length => 60),
```

Comment:
60 seems to be a bit too much:
![screenshot from 2016-03-23 17-24-25](https://cloud.githubusercontent.com/assets/1540531/13990238/4133faf0-f11c-11e5-8525-f1a30415fefa.png)